pkgs/applications/networking/feedreaders/feed2imap-go/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "feed2imap-go";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "Necoro";
+    repo = "feed2imap-go";
+    rev = "v${version}";
+    sha256 = "sha256-Ywo+ll6Ysy7H3bfzTBsT8U7M9XVOBlfd0v7RMgKoitY=";
+  };
+
+  vendorHash = "sha256-dtui0eChZXTrvprX3py/xCGmbHUYhk7nCHQ9pwBMHRY=";
+
+  postInstall = ''
+    # Move the print-cache tool out of the PATH, seems to be an internal development tool
+    mkdir $out/libexec
+    mv $out/bin/print-cache $out/libexec
+  '';
+
+  meta = with lib; {
+    description = "Uploads rss feeds as e-mails onto an IMAP server";
+    homepage = "https://github.com/Necoro/feed2imap-go";
+    license = licenses.gpl2;
+    platforms = platforms.linux;
```

Comment:
this seems like it is pure go, so it should work on darwin, too and we can just delete this line

Reply:
```suggestion
```